lib/jellyfish/room_service.ex

```diff
@@ -0,0 +1,72 @@
+defmodule Jellyfish.RoomService do
+  use GenServer
+  alias Jellyfish.Room
+
+  def start(init_arg, opts) do
+    GenServer.start(__MODULE__, init_arg, opts)
+  end
+
+  def start_link(opts) do
+    GenServer.start_link(__MODULE__, [], opts)
+  end
+
+  @impl true
+  def init(_) do
+    :ets.new(:rooms, [:protected, :set, :named_table])
```

Comment:
Why ETS?

Reply:
I think that the question here is whether RoomService might be overhelmed by HTTP requests. I would keep ets for now and observe how it behaves

Reply:
Ok, I thought that the `RoomService` process used the `:ets`, but it seems that it sort of replaces `GenServer.call/3`, which makes a lot more sense.